cloud/services/index.md

```diff
@@ -42,7 +42,7 @@ For additional documentation on how to:
 [hypertable-info]: /timescaledb/latest/how-to-guides/hypertables
 [time-bucket-info]: /timescaledb/latest/how-to-guides/query-data/advanced-analytic-queries#time-bucket
 [gap-filling-info]: /timescaledb/latest/how-to-guides/query-data/advanced-analytic-queries#gap-filling
-[aggregates-info]: /timescaledb/latest/getting-started/create-cagg
```

Comment:
Link to currently broken page:
https://docs.timescale.com/cloud/latest/services/#learn-more-about-timescaledb